### ISSUE TITLE
Question about cocos2d

### ISSUE THREAD

**Issue by IlluminatiWave** [bartlomiejduda/Tools#22]
The code is made for Cocos2d javascript, but does it work the same for Cocos2d Lua?

**Comment by IlluminatiWave** [comment]
Sorry, I was not specific

I meant if your plugin NEW Tools/Cocos2d/cocos2d_xxtea_script.js could work with the cocos2d-x version.

Since it seems to look the same, only instead of libcocos2djs.so , you use libcocos2dlua.so (the default xxtea function is the same, but in my so library they have left _Z13xxtea_decryptPhjS_jPj which calls xxtea_decrypt, or is it the other way around, I really don't know who calls who).

I have also changed the title which seems confusing

**Comment by bartlomiejduda** [comment]
Yeah, this script should be compatible with any cocos2d-x game that have "xxtea_decrypt" function.

There are also other methods of getting the key:
![obraz](https://github.com/bartlomiejduda/Tools/assets/5502283/bb22d696-2a5a-4cba-a63e-bc3c96fdbe63)

You can use any of these method to get the key and then you can use this tool https://github.com/bartlomiejduda/JSC-PyDecrypt-Tool to decrypt JSC files.
